CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     `producerGranuleId` identifier is set in updated CMR metadata
 
 ### Added
+- **CUMULUS-4080**
```

Comment:
Can you move the changelog down, order by ticket number?

Reply:
Sure but we should let folks know if we want to make that a standard. https://github.com/nasa/cumulus/pull/4021/commits/39fd3cf6f5fffc2642e4c4b7f74d89ff34f82a39